modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -36,6 +37,11 @@ def submit
 
         if form_is210966 && icn && first_party?
           handle_210966_authenticated
+        elsif params[:form_number] == '26-4555' && icn
+          parsed_form_data = JSON.parse(params.to_json)
+          form = SimpleFormsApi::VBA264555.new(parsed_form_data)
+          response = LGY::Service.new.post_grant_application(payload: form.as_payload)
+          render json: response.body if response.status == 201
```

Comment:
What happens when the response status isn't 201?

Reply:
Great point! I will change this to conform to the pattern we have elsewhere in this controller where we return the body and status regardless.